lib/mockexpress.js

```diff
@@ -112,12 +119,30 @@ var MockExpress = function(route) {
 				},
         status: function(status) {
           sideEffects.status = status;
-          callback(null, sideEffects);
-        },
+					return this;
+				},
+				sendStatus: function(status) {
+          sideEffects.status = status;
+					callback(null, sideEffects);
+				},
         end: function(end) {
           sideEffects.end = end;
           callback(null, sideEffects);
-        }
+				},
+				cookie: function(cookie) {
```

Comment:
The `res.cookie` function is actually fairly complicated, according to the [API](http://expressjs.com/en/api.html#res.cookie). It should take 3 arguments, `key`, `value` and `options`, the first two of which are required.

`sideEffects.cookie` should have, at minimum, both the `key` and `value`, preferably also the `options`. There should also be a test for this.